opm/material/common/UniformXTabulated2DFunction.hpp

```diff
@@ -42,6 +42,19 @@
 
 namespace Opm {
 
+
+
+/*!
+ * \brief Indicates how interpolation will be performed. Normal interpolation
+ *        is done by interpolating vertically between lines of sample points,
+ *        whereas LeftExtreme or RightExtreme implies guided interpolation, where
+ *        interpolation is done parallel to a guide line. With LeftExtreme
+ *        the lowest Y values will be used for the guide, and the guide line slope
+ *        extends unchanged to infinity. With RightExtreme, the highest Y values
+ *        are used, and the slope decreases linearly down to 0 at Y = 0.
```

Comment:
Yes it stays constant (no slope, normal interpolation) at y < 0. Comment changed. They are not symmetric because for LeftExtreme (PVTO) we can continue to use the same slope indefinitely since the domain (undersaturated) extends infinitely towards infinity, whereas for RightExtreme (PVTG) the undersaturated domain stops at y = 0.

Reply:
IMO it is a very serious mistake to hardcode the peculiarities of a specific use case into generic infrastructure. please add a scheme where the PVT[OG] requirements can be fulfilled by setting up the respective objects in a specific way, e.g., a constant up to which linear extrapolation shall used and after which a constant is returned can be specified for both ends.